projects/ui-components/shared/index.ts

```diff
@@ -0,0 +1 @@
+export * from './lf-shared-public-api';
```

Comment:
I don't think we want to export this?

Reply:
this is because we exported LoginState from shared. Where else can we import LoginState from?

Reply:
removed and it was working